src/main/webapp/i18n/de/cleanupService.json

```diff
@@ -15,13 +15,50 @@
             "deleteOldSubmissionVersions": "Alte Teilnahmeversionen löschen",
             "deleteOldFeedback": "Altes Feedback löschen"
         },
+        "tooltip": {
+            "deleteOrphans": "Löscht verwaiste Entitäten, die nicht mehr mit gültigen Ergebnissen oder Teilnahmen verbunden sind. Dazu gehören Feedback, Textblöcke und Bewertungen, die sich auf ungültige Ergebnisse, Teilnahmen oder Einreichungen beziehen.",
+            "deletePlagiarismComparisons": "Löscht Plagiatsvergleiche mit dem Status \"None\", die zu Kursen innerhalb des angegebenen Datumsbereichs gehören.",
```

Comment:
are we able to be more specific about comparisons with the status "None"? I.e., are those comparisons that found no plagiarism?

Reply:
done